SmartDeviceLinkTests/SDLStreamingMediaManagerSpec.m

```diff
@@ -4,13 +4,462 @@
+            });
+
+            it(@"should start both the audio and video stream managers with the protocol", ^{
+                OCMVerify([mockAudioLifecycleManager startWithProtocol:mockProtocol]);
+                OCMVerify([mockVideoLifecycleManager startWithProtocol:mockProtocol]);
+                expect(testStreamingMediaManager.audioStarted).to(beTrue());
+                expect(testStreamingMediaManager.videoStarted).to(beTrue());
+            });
+
+            it(@"should should not attempt to stop a current video or audio session", ^{
+                OCMReject([mockAudioLifecycleManager endAudioServiceWithCompletionHandler:[OCMArg any]]);
+                OCMReject([mockVideoLifecycleManager endVideoServiceWithCompletionHandler:[OCMArg any]]);
+            });
+        });
+
+        describe(@"stopping a running service on a transport", ^{
+            beforeEach(^{
+                OCMStub([mockVideoLifecycleManager endVideoServiceWithCompletionHandler:[OCMArg any]]).andDo(^(NSInvocation *invocation) {
+                    void (^handler)(void);
+                    [invocation getArgument:&handler atIndex:2];
+                    handler();
+                });
+
+                OCMStub([mockAudioLifecycleManager endAudioServiceWithCompletionHandler:[OCMArg any]]).andDo(^(NSInvocation *invocation) {
+                    void (^handler)(void);
+                    [invocation getArgument:&handler atIndex:2];
+                    handler();
+                });
+
+                [testStreamingMediaManager streamingServiceProtocolDidUpdateFromOldVideoProtocol:[OCMArg any] toNewVideoProtocol:nil fromOldAudioProtocol:[OCMArg any] toNewAudioProtocol:nil];
+            });
+
+            it(@"should stop both the audio and video stream managers", ^{
+                OCMVerify([mockAudioLifecycleManager endAudioServiceWithCompletionHandler:[OCMArg any]]);
+                OCMVerify([mockVideoLifecycleManager endVideoServiceWithCompletionHandler:[OCMArg any]]);
+                expect(testStreamingMediaManager.audioStarted).to(beFalse());
+                expect(testStreamingMediaManager.videoStarted).to(beFalse());
+            });
+
+            it(@"should tell the audio and video stream managers to destroy the protocol", ^{
+                OCMVerify([mockAudioLifecycleManager destroyProtocol]);
+                OCMVerify([mockVideoLifecycleManager destroyProtocol]);
+            });
+
+            it(@"should not attempt to start a new audio and video session", ^{
+                OCMReject([mockAudioLifecycleManager startWithProtocol:[OCMArg any]]);
+                OCMReject([mockVideoLifecycleManager startWithProtocol:[OCMArg any]]);
+            });
+
+            it(@"should notify the delegate object that the secondary transport can be destroyed", ^{
+                OCMVerify([mockSecondaryTransportDelegate destroySecondaryTransport]);
+            });
```

Comment:
These can be combined. Maybe the title could be "should stop both the audio and video stream managers"

Reply:
Combined.